src/index.test.tsx

```diff
@@ -0,0 +1,238 @@
+import { render } from '@testing-library/react';
+import React, { Fragment, PropsWithChildren } from 'react';
+import { Cell } from './Cell';
+import { HeaderRow } from './HeaderRow';
+import {
+  Column,
+  Columns,
+  DataTable,
+  Row,
+  Rows,
+  StringCell,
+  Table,
+  TableBody,
+  TableHeader,
+} from './index';
+import { useIndexContext } from './IndexContext';
+import { TableUIAdapterContextProvider } from './TableUIAdapterContext';
+
+const IndexCell = () => {
+  const index = useIndexContext();
+  return <Fragment>{index + 1}.</Fragment>;
+};
+
+const data = [
+  {
+    id: '1',
+    name: 'Gloves',
+    price: 544,
+    count: 5,
+    color: 'plum',
+  },
+  {
+    id: '2',
+    name: 'Salad',
+    price: 601,
+    count: 6,
+    color: 'turquoise',
+  },
+  {
+    id: '3',
+    name: 'Keyboard',
+    price: 116,
+    count: 1,
+    color: 'silver',
+  },
+];
+
+describe('ctablex', () => {
```

Comment:
- Add a test for a function `accessor` for `Column` component
- Add a test for `findColumns`
- Add a test for `ArrayOutput`
- Add a test for `getValue`